addons/discuss-button/addon.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "Discuss button",
-  "description": "Adds the discuss button to the navbar.",
+  "description": "Adds the discuss button to the navigation bar.",
```

Comment:
to the navigation bar... `on the website`?

Reply:
Eh, it's probably fine without "on the website" anyway

Reply:
The editor equivalent should be called "menu bar", not "navigation bar".